Packs/PaloAltoNetworks_IoT3rdParty/Playbooks/Bulk_Export_to_SIEM_-_PANW_IoT_3rd_Party_Integration_README.md

```diff
@@ -0,0 +1,29 @@
+This playbook gets all available assets ( alerts, vulnerabilities and devices) and send then to configured PANW third-party integration SIEM server.
```

Comment:
Gets all available assets (alerts, vulnerabilities, and devices) and sends them to the SIEM server for which you've configured PANW 3rd party integration. 

{Is it worth mentioning how the Syslog Sender integration is used to send data to the SIEM server in an agnostic data format that each SIEM can translate into their own data format?}